checkov/main.py

```diff
@@ -22,27 +22,49 @@
 from checkov.serverless.runner import Runner as sls_runner
 from checkov.terraform.plan_runner import Runner as tf_plan_runner
 from checkov.terraform.runner import Runner as tf_runner
+from checkov.helm.runner import Runner as helm_runner
 from checkov.version import version
 
 outer_registry = None
 
 logging_init()
+checkov_runner_module_names = ['cfn', 'tf', 'k8', 'sls', 'arm', 'tf_plan', 'helm']
+checkov_runners = ['cloudformation', 'terraform', 'kubernetes', 'serverless', 'arm', 'terraform_plan', 'helm']
 
+#Check runners for system deps, append --skip-framework list for failing deps.
+checkov_frameworks_unmatched_deps = []
+for runner in checkov_runner_module_names:
+    try:
+        globals()[f"{runner}_runner"]().system_deps
+    except:
+        continue
+    
+    if globals()[f"{runner}_runner"]().system_deps:
+            result = globals()[f"{runner}_runner"]().check_system_deps()
+            if result is not None:
+                checkov_frameworks_unmatched_deps.append(result)
 
-def run(banner=checkov_banner, argv=sys.argv[1:]):
+def run(banner=checkov_banner, runners=checkov_runners, argv=sys.argv[1:]):
```

Comment:
you can access checkov_runners directly - it will keep the method signature :)

Reply:
Thanks, and good catch, wasnt using `runners` was left behind from a refactor :) - Fixed.